src/apps/components/AppAlerts/SidebarAppAlert.tsx

```diff
@@ -0,0 +1,57 @@
+import { AppSections } from "@dashboard/apps/urls";
+import { ExclamationIcon } from "@dashboard/icons/ExclamationIcon";
+import { ExclamationIconFilled } from "@dashboard/icons/ExclamationIconFilled";
+import { Box, Text, Tooltip } from "@saleor/macaw-ui-next";
+import React, { useState } from "react";
+import { FormattedMessage } from "react-intl";
+import { Link } from "react-router-dom";
+
+import { useAppsAlert } from "./useAppsAlert";
+
+const ExclamationIconComponent = () => {
+  const [isHovered, setIsHovered] = useState(false);
+  const colorLighter = "#FFD87E";
+  const colorDefault = "#FFB84E";
```

Comment:
question: Wasn't possible to get those colours from the macaw theme? I'm asking because having hardcoded values will be harder to maintain.

Reply:
I couldn't find their macaw equivalents :/